docs/code-search/queries/index.mdx

```diff
@@ -15,7 +15,7 @@ This section documents the available search pattern syntax and interpretation in
 
 ### Keyword search (default)
 
-<Callout type="note" title="Beta">New in version 5.3: Keyword search is in the Beta stage and enabled by default. Toggle the "Keyword Search" switch to off to enable Standard search. Site administrators can disable Keyword search globally by setting `experimentalFeatures.keywordSearch: false` in site settings. Users can override the setting in their user settings. </Callout>
+<Callout type="note" title="Beta">New in version 5.3: Keyword search is in the Beta stage and enabled by default. Toggle the "Keyword Search" switch to off to enable Standard search. Site administrators can disable Keyword search globally by setting `experimentalFeatures.keywordSearch: false` in site settings. Users can override the setting in their user settings. With the keyword search feature enabled, use [patternType:keyword](/code-search/queries/language#pattern-type) in a search to turn that search into a keyword search.</Callout>
```

Comment:
I'd prefer to omit this sentence, since we don't want to encourage users to set `patterntype` directly in the search bar (which is what the rest of the note is referring to). Also, it will soon become the default, so even in Batch Changes and Code Monitors, users won't need to worry about setting it explicitly.